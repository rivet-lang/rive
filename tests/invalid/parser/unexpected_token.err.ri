fn main() { ] abc
invalid statement