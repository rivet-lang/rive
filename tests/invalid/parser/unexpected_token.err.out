tests/invalid/parser/unexpected_token.err.ri:1:12: error: expected `;`, but found token `]`
      | 
    1 | fn main() { ] abc
      |            ^     
error: could not compile `unexpected_token` module, aborting due to previous error